### PR TITLE
javascript finished

### PR DESCRIPTION
**Pull request by ganyfml**

### CODE REVIEW

File: static/js/home.js [hunk 1]
```diff
@@ -5,76 +5,77 @@
...
-                render_acc_detail(accinfo)});
-        }
-			// TODO, add click callback function to each marker
-		}
+  if(data.status != 0){
+    alert(data.data);
+    return;
+  }
+  console.log("get events");
+  acc_list = data.data;
```
Comment (wx1988) [inline static/js/home.js:54]: How about just call the function in render_markers.js

File: static/js/home.js [hunk 2]
```diff
@@ -5,76 +5,77 @@
...
+  for(var i=0;i < acc_list.length;i++){
+    pt_list[i] = new google.maps.LatLng(
+        acc_list[i].lat, acc_list[i].lng);
+    marker_list[i] = new google.maps.Marker({
+      position: pt_list[i],
+      map: map,
+      icon: '/static/imgs/red_cross_12.png'
+    });
+
+    makeInfoWindowEvent(map, infowindow, "test" + i, marker_list[i]);
```
Comment (wx1988) [inline static/js/home.js]: For the contentString parameter, how about replace it with the actual information as shown in step 3 of scenario 2. https://docs.google.com/document/d/1fmPTSg4u0wrEcPr8d2NBQoBAX0HVnGTnp5zt085a3zQ/edit

File: templates/segs_demo.html [hunk 3]
```diff
@@ -88,10 +101,13 @@
...
 function sigmoid(t) {
-    return 1/(1+Math.pow(Math.E, -t));
+	// re-scaled illustration
+    return 1/(1+Math.pow(Math.E, -0.1*t));
 }
 
+
 function geteventcb(data){
 	if(data.status != 0){
 		alert(data.data);
```
Comment (wx1988) [inline templates/segs_demo.html:113]: Replace the following code with the function call to the render_roadsegments function. 

Also add information window, with the information illustrated in step2 of scenario 2. 

https://docs.google.com/document/d/1fmPTSg4u0wrEcPr8d2NBQoBAX0HVnGTnp5zt085a3zQ/edit

File: static/js/home.js (hunk 1 above)
Comment (ganyfml) [inline static/js/home.js:54]: Actually I have the function in render_maker also.

File: static/js/home.js (hunk 2 above)
Comment (ganyfml) [inline static/js/home.js]: No problem. I will let you know after I finished.

File: static/js/render_roadsegments.js [hunk 4]
```diff
@@ -112,3 +129,70 @@ function HSVtoRGB(h, s, v)
...
+  6: 'Positive Barrier',
+  7: 'Parkland, Business',
+  8: 'Couplet',
+  9: 'Flexible Pos Barrier',
+  10: 'Striped',
+  11: 'Semi-Rigid Pos Barrier'
+}
+
+surface_type = {
+  00: '00 Primitive (Not Use don State System)',
```
Comment (wx1988) [inline static/js/render_roadsegments.js]: Remove number prefix in the string.

File: static/js/render_roadsegments.js [hunk 5]
```diff
@@ -112,3 +129,70 @@ function HSVtoRGB(h, s, v)
...
+  51: '51 Bituminous Surf Treatment on Topsoil',
+  52: '52 Bituminous Surf Treatment on Gravel or Stone',
+  60: '60 Mixed Bituminous, Non-Rigid Base',
+  61: '61 Mixed Bituminous, Rigid Base',
+  62: '62 Bituminous Penetration, Rigid Base',
+  63: '63 Bituminous Penetration, Non-Rigid Base',
+  65: '65 Sand Asphalt on Types Other Than 66, 67',
+  66: '66 Sand Asphalt on Bituminous Concrete',
+  67: '67 Bituminous Concrete',
+  '70-76': '70 Portland Cement Concrete',
```
Comment (wx1988) [inline static/js/render_roadsegments.js]: Expand '70-76' to integers

**Pull request merged**